ASP.NET Core/Data/AspNetCoreTemplate.Data.Common/AspNetCoreTemplate.Data.Common.csproj

```diff
@@ -16,7 +16,7 @@
     <PackageReference Include="StyleCop.Analyzers" Version="1.2.0-beta.113" PrivateAssets="all">
       <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
     </PackageReference>
-    <PackageReference Include="System.ComponentModel.Annotations" Version="4.5.0" />
+    <PackageReference Include="System.ComponentModel.Annotations" Version="4.7.0" />
```

Comment:
I don't think this is required anymore.

Reply:
It is required because of the `[Key]` attribute in `BaseModel<TKey>`